core/src/test/java/org/elasticsearch/common/geo/builders/MultiLineStringBuilderTests.java

```diff
@@ -68,6 +68,6 @@ static MultiLineStringBuilder mutate(MultiLineStringBuilder original) throws IOE
     }
 
     static MultiLineStringBuilder createRandomShape() {
-        return new MultiLineStringBuilder();
+        return MultiLineStringBuilder.class.cast(RandomShapeGenerator.createShape(random(), ShapeType.MULTILINESTRING));
```

Comment:
Interesting syntax, any reason not to cast using "(MultiLineStringBuilder)" here? I'm just curious ;-)

Reply:
I used this syntax because the `cast` method call explicitly calls `instanceof`. I'm not sure if compiler optimization does that for me with the cast operator? Maybe so, at which point this is just added verbosity?

Reply:
+1